contracts/transmuter/Storage.sol

```diff
@@ -92,6 +92,7 @@ struct DiamondStorage {
     bytes4[] selectors;                          // List of all available selectors
     mapping(bytes4 => FacetInfo) selectorInfo;   // Selector to (address, position in list)
     IAccessControlManager accessControlManager;  // Contract handling access control
+    uint8 statusReentrant;                        // If call is reentrant or not
```

Comment:
in fact I liked when it was in transmuter storage the reentrancy status because we could pack it in a slot with agToken and isRedemptionLive -> so more efficient gas wise than packing it with accesscontrolManager which is not read during a redemption